filenames.gypi

```diff
@@ -17,6 +17,7 @@
       'lib/browser/api/content-tracing.js',
       'lib/browser/api/dialog.js',
       'lib/browser/api/exports/electron.js',
+      'lib/browser/api/module-list.js',
```

Comment:
:+1: 

Forgot to update when I changed `exports/module-list.js` to `module-list.js`